crates/cheatnet/tests/cheatcodes/mod.rs

```diff
@@ -1,3 +1,4 @@
+mod deploy;
```

Comment:
I think we should make an issue for migrating `deploy` tests to `Cheatnet`.

Reply:
#569